cfgov/jinja2/v1/owning-a-home/explore-rates/index.html

```diff
@@ -308,7 +308,7 @@ <h3>Data table</h3>
                             <button class="chart-menu_btn
                                            cf-icon
                                            cf-icon__after
-                                           cf-icon-down">
+                                           cf-icon-download">
```

Comment:
Let's update or delete this too https://github.com/cfpb/cfgov-refresh/blob/master/test/unit_tests/apps/owning-a-home/js/explore-rates/RateCheckerChartMenu-spec.js#L27

Reply:
fixed in https://github.com/cfpb/cfgov-refresh/pull/4100/commits/70e143a4407fbde245f74d2c9023ab588016c378.